volttron/platform/messaging/topics.py

```diff
@@ -191,5 +191,11 @@
 WEATHER_BASE = 'weather'
 WEATHER_REQUEST = 'weather/request'
 
+NEW_WEATHER_BASE = 'weather2'
```

Comment:
Instead of NEW_WEATHER...how about using WEATHER2 ass it is more descriptive of what's going on